lib/i18n/payment-method-provider-type.js

```diff
@@ -23,6 +23,10 @@ const i18nTypes = defineMessages({
     id: 'ServiceBalance',
     defaultMessage: '{service} balance',
   },
+  [GQLV2_PAYMENT_METHOD_TYPES.CRYPTO]: {
+    id: 'Crypto',
+    defaultMessage: 'Crypto Contribution',
```

Comment:
Where is this appearing? Why is it "Crypto Contribution" and not just "Crypto"?

Reply:
This is appearing in the host dashboard; 

![image](https://user-images.githubusercontent.com/12435965/129055343-b576bfb9-2c86-43e2-8ce5-dc1f08247991.png)

I thought `Crypto Contribution` sounds better and more explicit; but I've changed it to `Crypto`. That seems fine too.